wd_helper.au3

```diff
@@ -1087,12 +1087,12 @@ EndFunc   ;==>_WD_ElementSelectAction
 
 ; #FUNCTION# ====================================================================================================================
 ; Name ..........: _WD_ElementStyle
-; Description ...: Set/Get element style property
-; Syntax ........: _WD_ElementStyle($sSession, $sElement, $sPropertyName, $sValue)
+; Description ...: Set/Get element style CSSProperty
+; Syntax ........: _WD_ElementStyle($sSession, $sElement, $sCSSProperty, $sValue)
 ; Parameters ....: $sSession - Session ID from _WD_CreateSession
 ;                  $sElement - Element ID from _WD_FindElement
-;                  $sPropertyName - style/CSS property name which should be set. When it is undefined (Default) then all properties are taken (array of properties will be returned)
-;                  $sValue - new property value to be set. When it is undefined (default), properties are retrieved instead of setting them
+;                  $sCSSProperty - style property name which should be set, when is not defined (Default) then all properties are taken (array of properties will be returned)
+;                  $sValue - new value to be set; current value will be retrieved when not supplied (Default)
 ; Return values .: Success - Response from web driver in JSON format ...... #TODO
```

Comment:
Please propose changes in Return values (Success/Failure) descriptions

Reply:
As for me this is the only place that should be reviewed.

Reply:
For me allmost all here is fine, only this description.
Please @Danp2  propose some changes.

Reply:
See 5f2b2b6f2f4b3a4fde6d3a705a7613e0c1b9e704

Reply:
Thanks for:
[Revise header](https://github.com/Danp2/au3WebDriver/pull/310/commits/5f2b2b6f2f4b3a4fde6d3a705a7613e0c1b9e704)

It looks good.